website/docs/r/resource_management_private_link.html.markdown

```diff
@@ -0,0 +1,61 @@
+---
+subcategory: "Management"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_resource_management_private_link"
+description: |-
+  Manages a Resource Management Private Link.
```

Comment:
can we expand on this

Reply:
changed to:
Manages a Resource Management Private Link to restrict access for managing resources in your Subscriptions.